source/exec.tex

```diff
@@ -7877,3 +7909,871 @@
 The storage referenced by \tcode{s} may be used by \tcode{*this}
 as temporary storage for the duration of the operation launched by this call.
 \end{itemdescr}
+
+\rSec2[exec.affine.on]{\tcode{execution::affine_on}}
```

Comment:
@dietmarkuehl I'm assuming this material is meant to go under "Coroutine utilities" (which happened to be at the end of [exec] prior to this meeting), rather than under "Namespace system_context_replaceability".

Reply:
yes, that is correct.

Reply:
No, I've got it. It was just that the edit instructions were not literally true ("at the end of [exec]" should instead of named the subclause at whose end you wanted it), so I mainly just want to confirm that I understood the intention.